decentraland/screens/BuyLandScreen.js

```diff
@@ -64,13 +69,15 @@ export class BuyLandScreen extends React.Component {
       // that catches the safeExecuteOrder successful event.
       await action.waitForNonceToUpdate();
 
+      updateMyAssetStatus(assetId, BOUGHT);
```

Comment:
I wonder about generalizing this term bought to something that could happen for any write tx. Like pending? And meta-tx might have state `BROADCAST` but not yet `PENDING`

Reply:
I'm using `BOUGHT` as a successful status. Maybe make more sense when we were using event listeners (i.e.: confs > 7)?

Reply:
Yep. But I'm saying since we might want to use this pattern for the MANA funding, for instance, where `BOUGHT` wouldn't describe the success state well.

That said, keeping an action-specific name for success might be better - I'm not sure.

Reply:
But yeah, we can wait until we're using event listeners more to formalize a standard around how to handle this stuff

Reply:
I think that this thread is related to the below one?

Reply:
Yep, i think so. So we'll use `PENDING`, `FAILED`, `CONFIRMED` or `SUCCESSFUL` for this instead of `BOUGHT`, etc.?

Reply:
Yes. I'm thinking to use `SUCCESSFUL` because it is a term less related to the blockchain. Make sense?

Reply:
Yep, sounds good. We'll use SUCCESSFUL for the current bought state after nonce updates.

And then whenever we track a single confirmation vs. 7, successful will be for 7 confirmations and something like `PROBABLY_SUCCESSFUL` will be for 1 confirmation

Reply:
Resolving this